charts/merlin/templates/_helpers.tpl

```diff
@@ -19,40 +19,14 @@
 {{- end -}}
 {{- end -}}
 
-{{- define "database.host" -}}
-{{- if .Values.postgresql.enabled }}
+{{- define "postgresql.host" -}}
+{{- if empty .Values.postgresql.postgresqlHost -}}
```

Comment:
Why not using `{{- if .Values.postgresql.enabled }}` ?

Reply:
to allow backward compatibility with current merlin deployment. 
In the current merlin deployment, the MLFlow DB is disabled, therefore if it is checking based on postgresql.enabled, this value won't be found out.

Rather than complicating the if in the helpers, checking whether postgresqlHost set up or not would be a cleaner way to use internal or ezternal.